app/models.py

```diff
@@ -182,3 +181,6 @@ class RequestInterest(models.Model):
         choices=STATUS,
         default='OPEN'
     )
+
+    def __str__(self):
+        return f'Interest from {self.user.display_name}'
```

Comment:
if you change the field name, don't forget to change the user to mentor or whatever field name you eventually decide on.